course_info/templates/course_info/partials/details.html

```diff
@@ -6,208 +6,256 @@ <h3 class="breadcrumbs">
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Meeting Time: </strong>
-                    </div>
-                    <div class="col-md-10">{{dc.courseInstance.meeting_time}}</div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Departments: </strong>
-                    </div>
-                    <div class="col-md-10" ng-repeat="dept in dc.courseInstance.departments">
-                         <div class="list-group" ng-if="dc.courseInstance.departments.length > 0">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="dept in dc.courseInstance.departments">
-                                {{dept.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Course Groups: </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="group in dc.courseInstance.course_groups">
-                                {{group.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Associated Official and Unofficial Site(s): </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="site in dc.courseInstance.sites">
-                                <div ng-switch on="site.map_type">
-                                    <div class="label label-primary col-md-1" ng-switch-when="official">Official</div>
-                                    <div class="label label-primary col-md-1" ng-switch-when="unofficial">Unofficial</div>
-                                </div>
-                                <div class="col-md-11"><a href="{{site.course_site_url}}"
-                                                          target="_blank">{{site.course_site_url}}</a>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+            <hu-editable-input
+                is-loading="dc.isDefined(dc.courseInstance.registrar_code_display)"
```

Comment:
Is there ever a case where `is-loading` isn't just `dc.isDefined(FORM_VALUE)`?  If not, can we just bake that logic into the directive instead of spelling it out every time?